### ISSUE TITLE
Could not create score database.

### ISSUE THREAD

**Issue by zetsyuko** [fluddokt/opsu#105]
**Version:** 0.15.0a
**OS:** Linux (aarch64)
**JRE:** The Android Project 0
BOARD: full_oppo6750_15331
FINGERPRINT: OPPO/A1601/A1601:5.1/LMY47I/1479785624:user/release-keys
HOST: WX-122-108
MODEL: A1601
INCREMENTAL: 1484100497
RELEASE: 5.1

**Error:** `Could not create score database.`
**Stack trace:**
```
java.sql.SQLException: android.database.sqlite.SQLiteException: no such table: info (code 1): , while compiling: INSERT OR IGNORE INTO info(key, value) VALUES('version', 20170201)
	at java.lang.reflect.Constructor.newInstance(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:288)
	at org.sqldroid.SQLDroidConnection.chainException(SQLDroidConnection.java:158)
	at org.sqldroid.SQLiteDatabase.execSQL(SQLiteDatabase.java:170)
	at org.sqldroid.SQLDroidStatement.execute(SQLDroidStatement.java:96)
	at itdelatrisu.opsu.db.ScoreDB.createDatabase(ScoreDB.java:189)
	at itdelatrisu.opsu.db.ScoreDB.init(ScoreDB.java:115)
	at itdelatrisu.opsu.db.DBController.init(DBController.java:50)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:202)
	at fluddokt.opsu.fake.GameOpsu.create(GameOpsu.java:206)
	at com.badlogic.gdx.backends.android.AndroidGraphics.onSurfaceChanged(AndroidGraphics.java:241)
	at android.opengl.GLSurfaceView$GLThread.guardedRun(GLSurfaceView.java:1564)
	at android.opengl.GLSurfaceView$GLThread.run(GLSurfaceView.java:1286)
Caused by: android.database.sqlite.SQLiteException: no such table: info (code 1): , while compiling: INSERT OR IGNORE INTO info(key, value) VALUES('version', 20170201)
	at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
	at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:908)
	at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:509)
	at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:588)
	at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:58)
	at android.database.sqlite.SQLiteStatement.<init>(SQLiteStatement.java:31)
	at android.database.sqlite.SQLiteDatabase.executeSql(SQLiteDatabase.java:1732)
	at android.database.sqlite.SQLiteDatabase.execSQL(SQLiteDatabase.java:1663)
	at org.sqldroid.SQLiteDatabase.execSQL(SQLiteDatabase.java:163)
	... 9 more
```

**Comment by zetsyuko** [comment]
How to fix these annoying script!!!

**Comment by fluddokt** [comment]
Should be fixed in 0.15.0b
